presto-hive/src/main/java/io/prestosql/plugin/hive/HiveMetadata.java

```diff
@@ -2538,6 +2539,10 @@ private static void validatePartitionColumns(ConnectorTableMetadata tableMetadat
                 .map(ColumnMetadata::getName)
                 .collect(toList());
 
+        if (allColumns.stream().anyMatch(x -> x.contains(" ") || x.contains(","))) {
+            throw new PrestoException(INVALID_COLUMN_NAME, "Hive does not support spaces and commas in column names");
```

Comment:
Do we want the error message to refer to Hive? Might be slightly confusing for end-users.